src/cache.rs

```diff
@@ -70,6 +70,8 @@ pub struct Cache {
     stdlib_ids: Option<Vec<FileId>>,
     /// The inferred type of wildcards for each `FileId`.
     wildcards: HashMap<FileId, Wildcards>,
+    /// Whether processing should try to continue even in case of errors.
+    error_tolerant: bool,
```

Comment:
Nitpick, but I think using a bespoke enum instead of booleans makes the intent clearer:

```rust
pub enum ParsingMode {
  ErrorTolerant,
  ErrorStrict,
}
...
    error_tolerant: ParsingMode
```

Then, when you read e.g. the code above in the LSP `Cache::new(ParsingMode::ErrorTolerant)`, you immediately know what is happening without having to jump to the Nickel codebase (which is a separate crate) and read the documentation of `Cache`. Cf "boolean blindness".